src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -128,33 +129,22 @@ public function collect(Request $request, Response $response, \Exception $except
             unset($this->controllers[$request]);
         }
 
-        if ($request->hasSession() && $request->getSession()->has('sf_redirect')) {
-            $this->data['redirect'] = $request->getSession()->get('sf_redirect');
-            $request->getSession()->remove('sf_redirect');
-        }
-
-        if ($request->hasSession() && $response->isRedirect()) {
-            $request->getSession()->set('sf_redirect', array(
-                'token' => $response->headers->get('x-debug-token'),
-                'route' => $request->attributes->get('_route', 'n/a'),
-                'method' => $request->getMethod(),
-                'controller' => $this->parseController($request->attributes->get('_controller')),
-                'status_code' => $statusCode,
-                'status_text' => Response::$statusTexts[(int) $statusCode],
-            ));
-        }
+        if (isset($session)) {
```

Comment:
Using `isset` on a variable should be avoided. I've fixed it in becdbd9ea8fc43c0400177eb6306d5aad63989ad